frontend/pages/DashboardPage/cards/MDM/MDM.tsx

```diff
@@ -117,14 +117,12 @@ const Mdm = ({
                   isLoading={isFetching}
                   defaultSortHeader={SOLUTIONS_DEFAULT_SORT_HEADER}
                   defaultSortDirection={DEFAULT_SORT_DIRECTION}
-                  hideActionButton
```

Comment:
if `actionButton` is undefined, it's already hidden, no need for an extra prop to hide it.

if it's a conditional, it uses `actionButton.hideButton` which already exists on `IActionButtonProps`